routes/web.php

```diff
@@ -58,9 +58,12 @@
 Route::namespace('Customer')->prefix('pages')->group(function () {
     Route::get('product/{id}', 'HomeController@show')->name('home.show');
     Route::post('/show-detail', 'HomeController@showDetail')->name('home.show_detail');
-    Route::get('/notification/{id}', 'Customer\HomeController@notification')->name('home.notification');
+    Route::get('/notification/{id}', 'HomeController@notification')->name('home.notification');
     Route::post('search', 'HomeController@search')->name('home.search');
     Route::post('search-detail', 'HomeController@searchDetail')->name('home.search_detail');
+    Route::get('orders', 'HomeController@order')->name('home.order');
+    Route::get('order-detail/{id}', 'HomeController@orderDetail')->name('home.order_detail');
+    Route::post('orders-cancel', 'HomeController@orderCancel')->name('home.order_cancel');
```

Comment:
```suggestion
    Route::post('cancel-order', 'HomeController@orderCancel')->name('home.order_cancel');
```
sửa những cái tương tự